src/servers/Scripts/quest/ManFst003.cpp

```diff
@@ -11,15 +11,240 @@
+   private:
+      // Basic quest information 
+      // Quest vars / flags used
+      // GetQuestBitFlag8
+      // GetQuestUI8AL
+      // GetQuestUI8BH
+      // GetQuestUI8BL
+      // GetQuestUI8CH
 
+      enum Sequence : uint8_t
+      {
+         Seq0 = 0,
+         Seq1 = 1,
+         SeqFinish = 255,
+      };
 
-public:
-   ManFst003() : EventScript( 65659 )
-   {}
+      // Quest rewards 
+      static constexpr auto RewardExpFactor = 100;
+      static constexpr auto RewardGil = 107;
+
+      // Entities found in the script data of the quest
+      static constexpr auto Actor0 = 1001140;
+      static constexpr auto Actor1 = 2;
+      static constexpr auto Actor2 = 1000197;
+      static constexpr auto Actor20 = 1000159;
+      static constexpr auto Actor3 = 1000768;
+      static constexpr auto Actor4 = 1000100;
+      static constexpr auto BindActor0 = 6229224;
+      static constexpr auto Item0 = 2000119;
+      static constexpr auto LocActor0 = 1003159;
+      static constexpr auto LocMarker01 = 2153091;
+      static constexpr auto LocMarker02 = 2153104;
+      static constexpr auto LocMarker03 = 2153111;
+      static constexpr auto LocMarker04 = 2154539;
+      static constexpr auto LocMarker05 = 2154540;
+      static constexpr auto LocMarker06 = 2154541;
+      static constexpr auto LocMarker07 = 2210446;
+      static constexpr auto LocMarker08 = 2210454;
+      static constexpr auto LocMarker09 = 2210461;
+      static constexpr auto LocMotion0 = 799;
+      static constexpr auto Poprange0 = 2280858;
+      static constexpr auto RewardDesion = 1;
+      static constexpr auto Seq0Actor0 = 0;
+      static constexpr auto Seq0Actor0Lq = 50;
+      static constexpr auto Seq1Actor0 = 4;
+      static constexpr auto Seq1Actor1 = 1;
+      static constexpr auto Seq1Actor1Wait = 51;
+      static constexpr auto Seq1Actor2 = 2;
+      static constexpr auto Seq1Actor3 = 3;
+      static constexpr auto Seq1Actor3Npctradeno = 99;
+      static constexpr auto Seq1Actor3Npctradeok = 100;
+      static constexpr auto Seq2Actor4 = 5;
+      static constexpr auto Territorytype0 = 132;
+      static constexpr auto UnlockDesion = 14;
+
+   public:
+      ManFst003() : EventScript( 65659 ){}; 
+      ~ManFst003(){}; 
 
    void onTalk( uint32_t eventId, Entity::Player& player, uint64_t actorId ) override
    {
+      auto actor = Event::mapEventActorToRealActor( static_cast< uint32_t >( actorId ) );
+
+      if( actor == Actor0 )
+      {
+         Scene00000( player );
+      }
+      else if( actor == Actor1 )
+      {
+         player.eventActionStart( 0x050002, 0x13,
+            [&]( Entity::Player& player, uint32_t eventId, uint64_t additional )
+         {
+            player.sendQuestMessage( 0x050002, 0, 1, 0, 0 );
+            player.registerAetheryte( 2 );
+            player.learnAction( 1 );
+            Scene00051( player );
+         },
+            [&]( Entity::Player& player, uint32_t eventId, uint64_t additional )
```

Comment:
nullptr should work for empty function bodies instead of defining a lambda

Reply:
I modify it, look if it's ok now